packages/netlify-cms-core/src/redux/index.ts

```diff
@@ -13,7 +13,7 @@ declare global {
 }
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-const store = createStore<State, any, {}, {}>(
+const store = createStore<any, any, {}, {}>(
   createRootReducer(history),
```

Comment:
You can use `(createRootReducer(history) as unknown) as Reducer<State>` and keep `createStore<State`